serializers/src/glTF/2.0/glTFMaterialExporter.ts

```diff
@@ -181,9 +181,33 @@ export class _GLTFMaterialExporter {
             }
         }
 
+        // Do a naive check for materials that contain extensions that have texture-related members.
+        if (material.extensions) {
+            for (let extension in material.extensions){
+                let extensionObject = material.extensions[extension];
+                for (let member in extensionObject){
+                    if (member.indexOf("Texture") !== -1){
+                        if (extensionObject[member]){
+                            return true;
+                        }
+                    }
+                }
+            }
+        }
+
         return false;
     }
 
+    public _getTextureInfo(babylonTexture: Nullable<BaseTexture>): Nullable<ITextureInfo> {

```

Comment:
Its actually a little strange that other extensions like `KHR_materials_sheen` populate their own list of exported textures, that doesn't seem to scale well when we get to a point where materials may have multiple shared textures used across multiple extensions for the same material. Not a huge concern now, but something I want to make sure I take a note of.

Reply:
Refactored sheen to be less strange, reused my material exporter helper functions to simplify how to get the textureInfo for a given texture this extension adds.